week-2/debug/0.js

```diff
@@ -5,3 +5,5 @@ function multiply(a, b) {
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+
+/* It will multiply A and B
```

Comment:
The goal of this task is to find the mistake and correct it - what do you think it is ? Have you run the code?